docs/developer-docs/setup/cycles/cycles-ledger.md

```diff
@@ -0,0 +1,158 @@
+
+```
+dfx identity whoami
+```
+
+If you're using the default identity, you should create a new identity with the command:
+
+```
+dfx identity new IDENTITY_NAME
+```
+
+Replace `IDENTITY_NAME` with the an identity name of your choice, such as 'MyIdentity`.
+
+Your identity's seed phrase will be returned. Be sure to save this in a secure location. Then, set this identity to be used by the current session:
+
+```
+dfx identity use IDENTITY_NAME
+```
+
+Then, to get the account ID for this identity, use the command:
+
+```
+dfx ledger account-id
+```
+
+This will display your account number on the ICP ledger. It looks similar to this:
+
+```
+e213184a548871a47fb526f3cba24e2ee2fbbc8129c4ab497ef2ce535130a0a4
+```
+
+Then, to return the principal ID associated with your identity, run the command:
+
+```
+dfx identity get-principal
+```
+
+This principal ID will be used when others want to send you cycles using the cycles ledger. 
+
+## Using the cycles ledger
+
+### Checking your cycles balance
+
+To check the cycles balance of your user principal ID, use the command:
+
+```
+dfx cycles balance
+```
+
+### Converting ICP to cycles
+
+To convert ICP into cycles using the cycles ledger, use the command:
+
+```
+dfx cycles convert --icp-amount AMOUNT
+```
+
+Replace `AMOUNT` with the number of ICP to convert into cycles, such as `2.7`.
+
+### Transferring cycles to a principal
+
+To transfer cycles to another principal ID, use the command:
+
+```
+dfx cycles transfer AMOUNT --to-account PRINCIPAL_ID
+```
+
+Replace `AMOUNT` with the number of cycles, such as `34000000`, and `PRINCIPAL_ID` with the principal ID you'd like to transfer the cycles to, such as `tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe`.
+
+### Topping up a canister
+
+To transfer cycles into a canister to top it up, use the command:
+
+```
+dfx cycles transfer `AMOUNT` --top-up `CANISTER_ID`
+```
+
+Replace `AMOUNT` with the number of cycles, such as `34000000`, and `CANISTER_ID` with the canister ID you'd like to transfer the cycles to, such as `gastn-uqaaa-aaaae-aaafq-cai`.
+
+### Creating a canister using cycles
+
+New canisters can be created using the cycles ledger by using the `dfx canister create` and `dfx deploy` commands.
+
+Both of these commands have the following options and syntax to create a new canister:
+
+```
+dfx canister create CANISTER_NAME
+dfx deploy CANISTER_NAME
+```
+
+Replace `CANISTER_NAME` with a name for the new canister.
+
+With these two commands, there are two **optional** flags that can be used:
+
+- `dfx create canister CANISTER_NAME --next-to CANISTER_ID:` The new canister will be installed on the same subnet as the canister provided.
+- `dfx create canister CANISTER_NAME --subnet SUBNET_ID`: The new canister will be installed on the given subnet.
+
+If neither flag is used, the canister will be created on a random subnet using the user principal’s cycles.
```

Comment:
@sesi200, we may have to update this part, depending on the outcome of the architecture meeting.

Reply:
How it works now:

- if `--subnet`, `--subnet-type`, or `--next-to` is specified, those are used
- if nothing is specified and no canisters exist yet in that project, a random subnet is used
- if nothing is specified and the project contains canisters on only one subnet, then that subnet is used
- if nothing is specified and the project contains canisters on multiple subnets, then dfx asks the user to specify a subnet